tests/performance/local/condition_variable_overhead.cpp

```diff
@@ -0,0 +1,233 @@
+//  Copyright (c) 2024 ETH Zurich
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <pika/config.hpp>
+#include <pika/execution.hpp>
+#include <pika/init.hpp>
+#include <pika/modules/timing.hpp>
+#include <pika/runtime.hpp>
+#include <pika/testing/performance.hpp>
+#include <pika/thread.hpp>
+
+#include <fmt/format.h>
+#include <fmt/printf.h>
+
+#include <cstdint>
+#include <cstdlib>
+#include <iostream>
+#include <limits>
```

Comment:
```suggestion
#include <limits>
#include <vector>
```
https://app.circleci.com/pipelines/github/pika-org/pika/11001/workflows/f70ecfb2-0d3b-43af-bbf1-3cf868e31979/jobs/115533/artifacts